service/controller/v1/prometheus/prometheus.go

```diff
@@ -62,12 +68,24 @@ var (
 	// ClusterTypeLabel is the label used to hold the cluster's type.
 	ClusterTypeLabel = "cluster_type"
 
+	// NamespaceLabel is the label used to hold the application's namespace.
```

Comment:
Should be `ExportedNamespaceLabel is the label used to hold the application's namespace.`